### ISSUE TITLE
"MyApp" Sample App doesn't work when converted to a Console Application

### ISSUE THREAD

**Issue by Sharpiro** [Squirrel/Squirrel.Windows#967]
I was able to take the "MyApp" sample from the getting started guide, add:
```
using (var mgr = new UpdateManager("C:\\Projects\\MyApp\\Releases"))
{
    await mgr.UpdateApp();
}
```
And have everything work out of the box.

However, I was struggling re-creating my own implementation where my application was instead a Console Application, and I couldn't get it working.  

Even when I took the sample "MyApp" from the tutorial, and converted it to a Console App in the properties window with 1 click, it would immediately fail after nuget packaging, and then install.  

Even trying to kick it off on a separate thread failed to work as a Console App:
```
private static void Main(string[] args)
{
    Task.Run(async () =>
    {
        using (var mgr = new UpdateManager(@"C:\gitbase\MyApp\releases"))
        {
            Console.WriteLine("Starting update...");
            await mgr.UpdateApp();
            Console.WriteLine("Completed update...");
        }
    });

    var assembly = Assembly.GetExecutingAssembly();
    var fvi = FileVersionInfo.GetVersionInfo(assembly.Location);
    Console.WriteLine(fvi.FileVersion);
    Console.ReadKey();
}
```

Basically, it's like the process is unable to fully complete as a console application, despite the "Completed update" line being successfully hit.  ```%LocalAppData%\MyApp``` is created, but either no ```MyApp.exe``` is created in that folder, or it gets created w/ a file size of 0 Bytes and windows sees it as invalid, and the desktop shortcut that is auto-created will of course fail since it's pointing to a non-existent ```%LocalAppData%\MyApp\MyApp.exe```

**Comment by ralsu091** [comment]
The logs should give you more information:

> How can I tell what is going wrong with the releasify?
Check packages\Squirrel.Windows.VERSION\tools\SquirrelSetup.log for logging information when creating packages.

**Comment by billw2012** [comment]
It appears as though the new stub exe stuff simply doesn't work with console apps. See #945.

**Comment by gojanpaolo** [comment]
@shiftkey dup of #1098
Issue still exists with latest version of Squirrel
